docs/docsite/rst/scenario_guides/vmware_scenarios/vmware_requirements.rst

```diff
@@ -60,3 +60,5 @@ For example, if ``/var/vmware/certs/vcenter1.crt`` is the SSL certificate for yo
        password: '{{ vcenter_password }}'
      environment:
        REQUESTS_CA_BUNDLE: /var/vmware/certs/vcenter1.crt
+
+There is a `known issue <https://github.com/psf/requests/issues/3829>`_ in ``requests`` library (version 2) which you may want to consider while using this environment variable. Basically, setting ``REQUESTS_CA_BUNDLE`` environment variable on managed nodes overrides ``validate_certs`` value. This may result in unexpected behaviour while running the playbook. Please see https://github.com/ansible-collections/community.vmware/issues/601 and https://github.com/vmware/vsphere-automation-sdk-python/issues/254 for more information.
```

Comment:
```suggestion
There is a `known issue <https://github.com/psf/requests/issues/3829>`_ in ``requests`` library (version 2) which you may want to consider when using this environment variable. Basically, setting ``REQUESTS_CA_BUNDLE`` environment variable on managed nodes overrides the ``validate_certs`` value. This may result in unexpected behavior while running the playbook. Please see https://github.com/ansible-collections/community.vmware/issues/601 and https://github.com/vmware/vsphere-automation-sdk-python/issues/254 for more information.
```